### DISCUSSION TITLE
Add/edit events on local ICS files

### DISCUSSION THREAD

**Discussion by sbot50** [bitfireAT/icsx5#90] [Ideas]
this might be a stupid question, but literally whats the point of this app if you cannot edit local ICS files. every calendar app in existance nowadays can import ICS files. I need a way to add events to the ics files, since i have yet to find anything on android that does so.

Its simple, i just want my calendar to be on my phone and my pc and be able to edit the events on it.

syncthing works great for the keeping files on my devices and syncing them.
this app... well i thought i would be able to add events to ics files with it but appearently not, still no clue what the use of this app is.

anyways now ill just sit here and probably be bashed in the comments.

**Comment by rfc2822** [comment]
> this app... well i thought i would be able to add events to ics files with it but appearently not, still no clue what the use of this app is.

Quoted from the ICSx5 homepage:

> ICSx⁵ is an Android app to subscribe to remote (“Webcal”) or local iCalendar files (like time tables of your school/university or event files of your sports team). It will
> - download each iCalendar file regularly at a given interval and
> - import it into a native read-only calendar, which then can be used with your favorite calendar app.

I have subscribed holiday calendars and sports and other community events from various sources and thus always have up-to-date events on my phone. So that's the purpose :)

> every calendar app in existance nowadays can import ICS files.

ICSx5 doesn't import, but synchronize: it imports regularly and updates the events. For instance, new events will automatically be added.

If you don't need regular synchronization, of course it makes more sense to just import the ICS with a calendar app instead of using ICSx5.

I understand the use-case of creating/updating Webcals, but that's currently not within the scope of ICSx5.
